pkg/webhook/parser.go

```diff
@@ -130,6 +134,9 @@ func (c Config) ToMutatingWebhook() (admissionreg.MutatingWebhook, error) {
 		MatchPolicy:   matchPolicy,
 		ClientConfig:  c.clientConfig(),
 		SideEffects:   c.sideEffects(),
+		// TODO(jiachengxu): AdmissionReviewVersions becomes required in admissionregistration/v1, here we default it
+		// to `v1`, and we should support to config the `AdmissionReviewVersions` as a marker.
+		AdmissionReviewVersions: []string{defaultVersion},
```

Comment:
I can implement this `AdmissionReviewVersions` as a marker by following a PR, don't want to do this in this PR since this PR is already large though :)

Reply:
This should probably include "v1beta1" since controller-runtime uses v1beta1 admission reviews right?

Reply:
@estroz Yeah, you are right, we should add `v1beta1` here also.